AspNetCore/DotvvmAuthSample/DotvvmAuthSample/DotvvmAuthSample.csproj

```diff
@@ -24,10 +24,10 @@
         <Folder Include="wwwroot\" />
     </ItemGroup>
     <ItemGroup>
-        <PackageReference Include="Microsoft.AspNetCore" Version="2.0.3" />
-        <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.3" />
-        <PackageReference Include="DotVVM.AspNetCore" Version="2.0.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="2.0.0" />
+        <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
```

Comment:
I think that from .NET Core 3.0, the references to `Microsoft.AspNetCore.*` are not needed anymore - they should be already referenced by the `Microsoft.NET.Sdk.Web`. 
Can you please remove all three packages (keep only `DotVVM.AspNetCore` and try whether it works?

Reply:
Microsoft.AspNetCore.Authentication.OpenIdConnect seems to be required. The sample app works without them.